README.md

```diff
@@ -39,6 +40,18 @@ See [Release Notes](RELEASE_NOTES.md) and [System Requirements](SYSTEM_REQUIREME
 ## Installation 
 See [Installation from Sources](INSTALL.md) to learn how to install oneTBB. 
 
+## Governance
+
+The oneTBB project is governed by the UXL Foundation and you can get involved in this project in multiple ways. 
+It is possible to join the [Language Special Interest Group (SIG)](https://github.com/uxlfoundation/foundation/blob/main/language) meetings
+where the groups discuss and demonstrate work using this project. Members can also join the Open Source and Specification Working Group meetings.
```

Comment:
```suggestion
to discuss and demonstrate work using this project. 
* Join the Open Source and Specification Working Group meetings.
```

Reply:
Not clear how users can join open source and specification meetings. Is it also a part of the SIG meetings?